sounds/templatetags/sound_signature.py

```diff
@@ -0,0 +1,18 @@
+import urlparse
+from django.urls import reverse
+from django.contrib.sites.models import Site
+from django import template
+
+register = template.Library()
+
+
+@register.filter(name='sound_signature_replace')
+def sound_signature_replace(value, sound):
+    domain = "https://%s" % Site.objects.get_current().domain
+    abs_url = urlparse.urljoin(domain, reverse('sound', args=[sound.user.username, sound.id]))
+
+    replace = [("$$SOUND_ID$$", str(sound.id)),
+            ("$$SOUND_URL$", abs_url)]
```

Comment:
The placeholders thing is a nice idea :) 
I agree sound id it can be useful as as Bram commented https://github.com/MTG/freesound/pull/1007#issuecomment-341431636.